sections/futures/MobileTrade/drawers/TradePanelDrawer.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+import FullScreenModal from 'components/FullScreenModal';
+import { zIndex } from 'constants/ui';
+import TradeIsolatedMargin from 'sections/futures/Trade/TradeIsolatedMargin';
+
+type TradePanelDrawerProps = {
+	open: boolean;
+	closeDrawer(): void;
+};
+const TradePanelDrawer: FC<TradePanelDrawerProps> = ({ open, closeDrawer }) => {
+	return (
+		<StyledModal isOpen={open} onDismiss={closeDrawer}>
+			<Background>
+				<Closer onClick={closeDrawer} />
+				<Foreground>
+					<TradeIsolatedMargin mobile />
+				</Foreground>
+			</Background>
+		</StyledModal>
+	);
+};
+
+const StyledModal = styled(FullScreenModal)`
+	top: 0;
+	background-color: transparent;
+	z-index: ${zIndex.DRAWER};
+
+	& > div {
```

Comment:
Are these styled just being applied to direct div child which is Background, so we could add them to that instead? Prefer to avoid the nested css styles where possible

Reply:
It's actually being added to DialogOverlay, which is indirectly the first child because of the structure of the BaseModal component. We can't target it directly from here, and extending the component itself might have too many negative externalities. We probably should make a more dumb/barebones base modal component that isn't too opinionated because of all the new styles we're getting.

Reply:
ok got it, another one for the refactor list